zmq/src/caml_zmq_stubs.c

```diff
@@ -60,8 +60,8 @@ CAMLprim value caml_zmq_version(value unit) {
  * Init
  */
 
-CAMLprim value caml_zmq_new(value num_threads) {
-    CAMLparam1 (num_threads);
```

Comment:
This value is unused. In the old days creating a context required a number of io threads as argument, but the zmq_ctx_new does not. I guess the naming is an artifact from using zmq_ctx_init

Reply:
I'm almost certain that's correct regarding where the name comes from.